<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en">

<head>

	<meta charset="utf-8" />
	<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
	
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
		<meta name="robots" content="index, follow" />
		<link rel="apple-touch-icon" sizes="167x167" href="https://riveraomics.github.io/resources/OMICS_logo.png" />
<link rel="apple-touch-icon" sizes="180x180" href="https://riveraomics.github.io/resources/OMICS_logo.png" />
<link rel="apple-touch-icon" sizes="152x152" href="https://riveraomics.github.io/resources/OMICS_logo.png" />
<link rel="mask-icon" href="https://riveraomics.github.io/resources/OMICS_logo.png" color="rgba(255,255,255,1.00)" /><link rel="icon" type="image/png" href="https://riveraomics.github.io/resources/OMICS_logo.png" sizes="32x32" />
<link rel="icon" type="image/png" href="https://riveraomics.github.io/resources/OMICS_logo.png" sizes="16x16" />
<link rel="icon" type="image/png" href="https://riveraomics.github.io/resources/OMICS_logo.png" sizes="64x64" />

	<meta name="twitter:card" content="summary">
	<meta name="twitter:title" content="Breaking Boundaries of Biomedical Research in Puerto Rico with 10x Genomics">
	<meta name="twitter:image" content="https://riveraomics.github.io/resources/CCC-UPR-10X-SYMP-DET.jpg">
	<meta name="twitter:url" content="https:/riveraomics.github.io/blog/files/breaking-boundaries-biomedical-research-puerto-rico-with-10x-genomics.html">
	<meta property="og:type" content="website">
	<meta property="og:site_name" content="OMICS">
	<meta property="og:title" content="Breaking Boundaries of Biomedical Research in Puerto Rico with 10x Genomics">
	<meta property="og:image" content="https://riveraomics.github.io/resources/CCC-UPR-10X-SYMP-DET.jpg">
	<meta property="og:url" content="https:/riveraomics.github.io/blog/files/breaking-boundaries-biomedical-research-puerto-rico-with-10x-genomics.html">
	

	<title>Breaking Boundaries of Biomedical Research in Puerto Rico with 10x Genomics | Blog | OMICS</title>
	<link rel="stylesheet" type="text/css" media="all" href="../../rw_common/themes/Future/consolidated-29.css?rwcache=730756410" />
		
	
	 <style type="text/css" media="all">.hero,
.hero-content {
	height: 77vh;
	}
.hero-background {
	height: 70vh;
	}

.hero-title em {
    font-size: 0.5em;
}


@media (min-width: 991px) { .nav-item.has-dropdown .dropdown-menu {
    background-color: black !important;
    border: 1px solid white !important;
}}

@media (max-width: 991px) { .navbar .navbar-nav {
    background-color: solid black !important;
    border: 1px solid white !important; 
}}

.dropdown-menu .nav-item .nav-link {
      color: white !important;
    }
.dropdown-menu .nav-item .nav-link:hover {
      color: rgba(247,0,12,1) !important;
    }
    

.blog-entry-title {
  font-size: 44px;
}

@media (max-width: 800px) {.blog-entry-title {
  font-size: 33px;
}}

blockquote {
  font-size: 33px; 
  line-height: 1.5;
  }
  
@media (max-width: 800px) {blockquote {
  font-size: 22px; 
  line-height: 1.5;
  }}

span[style*="Hiragino Kaku Gothic Pro"],
span[style*="ヒラギノ角ゴ Pro W3"],
span[style*="Osaka"],
span[style*="ＭＳ Ｐゴシック"],
span[style*="MS PGothic"]  span[style*=monospace] {
    font-size: 17px !important; 
}

@media (max-width: 800px) {
	span[style*="Hiragino Kaku Gothic Pro"],
span[style*="ヒラギノ角ゴ Pro W3"],
span[style*="Osaka"],
span[style*="ＭＳ Ｐゴシック"],
span[style*="MS PGothic"] , span[style*=monospace] {
    font-size: 15px !important;
}}

/* Basic styling */
.button-container {
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
  align-items: center;
}

.zoom-button {
  display: inline-block;
  width: 1000px;
  height: 200px;
  position: relative;
  overflow: hidden;
  border-radius: 5px;
  text-align: center;
  background-size: cover;
  background-position: left;
  }

.overlay {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background: linear-gradient(rgba(255, 0, 0, 0.5), rgba(0, 0, 0, 1));
  color: #FFFFFF;
  opacity: 0;
  transition: opacity 0.2s;
  display: flex;
  justify-content: center;
  align-items: center;
}

.button-text {
  font-size: 24px;
  color: #fff;
  opacity: 1;
  transform: scale(1);
  transition: transform 0.3s, opacity 0.3s;
  position: relative;
  z-index: 1;
}

.button-text-df {
  font-size: 33px;
  background-color: rgba(0, 0, 0, 0.7); /* 50% black alpha */
    padding: 10px;
  color: #fff;
  opacity: 1;
  transform: scale(1);
  transition: transform 0.3s, opacity 0.3s;
  position: relative;
  z-index: 1;
}

/* Zoom effect on hover */
.zoom-button:hover {
  transform: scale(1.1);
}

.zoom-button:hover .overlay {
  opacity: 1;
}

.zoom-button:hover .button-text {
  transform: scale(1.2); /* Scale text on hover */
}

.author-section {
  display: flex;
  align-items: center;
}

.author-image {
  flex: 0 0 auto;
  margin-right: 20px;
}

.author-image img {
  width: 80px;
  height: 80px;
}



</style> <script>
			 <script src="https://cdnjs.cloudflare.com/ajax/libs/showdown/2.0.3/showdown.min.js"></script>

<script>
  // Function to fetch Markdown content from GitHub
  function fetchMarkdownFromGitHub() {
    const githubMarkdownURL = 'https://raw.githubusercontent.com/riverainitiatives/library/main/library_markdown.md'; // Replace with your GitHub URL
    fetch(githubMarkdownURL)
      .then(response => response.text())
      .then(markdownContent => {
        const converter = new showdown.Converter();
        const htmlContent = converter.makeHtml(markdownContent);

        // Update the Moodle Wiki page with HTML content
        document.getElementById('wiki-content').innerHTML = htmlContent;
      })
      .catch(error => {
        console.error('Error fetching Markdown:', error);
      });
  }

  // Call the function to fetch and display the Markdown content
  fetchMarkdownFromGitHub();
</script> 
		</script> <link rel="alternate" type="application/rss+xml" title="Lab to Market" href="https://riveraomics.github.io/blog/files/feed.xml" />
<script type="text/javascript" async src="https://riveraomics.github.io/blog/files/meta.js"></script>

</head>

<!-- This page was created with RapidWeaver from Realmac Software. http://www.realmacsoftware.com -->

<body>
	<nav class="navbar navbar-expand-lg">
		<div class="container">
			<a class="navbar-brand" href="https://riveraomics.github.io/"><img src="../../rw_common/images/omics_group_sq3 copy.png" width="766" height="247" alt="Lab to Market"/> <span class="navbar-title">Lab to Market</span></a>
			<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false"
			 aria-label="Toggle navigation">
				<span class="navbar-toggler-icon"></span>
			</button>
			<div class="collapse navbar-collapse" id="navbarNav">
				<ul class="navbar-nav ml-auto"><li class="nav-item"><a href="../../team/" rel="" class="nav-link">Team</a></li><li class="nav-item"><a href="../../projects/" rel="" class="nav-link">Research</a></li><li class="nav-item"><a href="../../publications/" rel="" class="nav-link">Publications</a></li><li class="nav-item"><a href="../../protocols/" rel="" class="nav-link">Protocols</a></li><li class="nav-item"><a href="https://github.com/riveraomics" target="_blank" class="nav-link">Github</a></li><li class="nav-item has-dropdown"><a href="../../intranet/" rel="" class="nav-link dropdown-toggle" data-toggle="dropdown">Intranet</a><ul class="navbar-nav ml-auto"><li class="nav-item"><a href="../../intranet/activities-null/" rel="" class="nav-link">Activities</a></li><li class="nav-item"><a href="../../intranet/library/" rel="" class="nav-link">Library</a></li><li class="nav-item"><a href="https://omics.riverainc.org/moodle/login/index.php" rel="" class="nav-link">Academy</a></li><li class="nav-item"><a href="https://workspace.google.com/dashboard" rel="" class="nav-link">Workspace</a></li><li class="nav-item"><a href="" rel="" class="nav-link">Boquerón HPC</a></li></ul></li><li class="nav-item active"><a href="../" rel="" class="nav-link">Blog</a></li></ul>
			</div>
		</div>
	</nav>

	<header class="hero" id="hero">
        <div class="hero-background"></div>
		<div class="hero-overlay"></div>
		<h1 class="hero-title">
			Lab to Market
			<em>Bridging Science and Business</em>
		</h1>
	</header>

	<div class="content">
		<section class="main" style="position: relative;">
			<div class="container">
                <div class="row">
                    <div class="col-sm-12 main">
                        	
	<div class="blog-archive-entries-wrapper">
		<div id="unique-entry-id-1" class="blog-entry"><h1 class="blog-entry-title">Breaking Boundaries of Biomedical Research in Puerto Rico with 10x Genomics</h1><div class="blog-entry-date">December 23, 2023  - Authored by: <span class="blog-entry-category"><a href="category-paola-n.-aponte-castro.html">Paola N. Aponte-Castro</a></span><span class="blog-entry-category"> | <a href="category-osvaldo-d.-rivera-gonz00e1lez.html">Osvaldo D. Rivera-Gonz&aacute;lez</a></span></div><div class="blog-entry-body"><br /><a href="https://www.10xgenomics.com/" target="_blank">10X Genomics</a> (<a href="https://investors.10xgenomics.com/stock-info/">NASDAQ:TXG</a>), the current industry leader of single-cell genomics solutions, has recently been noticeably active in Puerto Rico, hosting two distinct activities in the San Juan metropolitan area; a genomics workshop sponsored by the <a href="https://cicim.upr.edu/" target="_blank">Molecular Sciences Research Center (MSRC)</a> and a Single Cell Genomics Symposium at the <a href="https://www.cccupr.org/investigacion/" target="_blank">Comprehensive Cancer Center of the University of Puerto Rico (CCC-UPR)</a>. The activities collectively highlighted the significance of comprehensive biological insights, stressing the need to understand the molecular biology of single cells to paint a more accurate picture of the natural complexity of tissues. The symposiums and workshops were centered on 10x Genomics' cutting-edge platforms&mdash;Chromium Single Cell, Visium Spatial, and Xenium In Situ&mdash; their unique contribution to emerging biological systems experimentation, namely to studying cell variations, adaptive immunity, biomarker detection, cell-specific epigenetic mapping, and spatial analysis of gene expression within tissue sections, all of which enable deeper context to cellular interactions. <br /><br /><img class="imageStyle" alt="msrc_ccc" src="msrc_ccc.png" width="3666" height="2051" /><br /><span style="font:12px &#34;Hiragino Kaku Gothic Pro&#34;,&#34;ヒラギノ角ゴ Pro W3&#34;,&#34;Osaka&#34;,&#34;ＭＳ Ｐゴシック&#34;,&#34;MS PGothic&#34;, sans-serif; color:#7F8080;">Photo: The MSRC (left) and UPR-CCC (right) where 10X Genomics has hosted their recent activities.<br /></span><hr><br />Before further diving into the article, let me quickly summarize the 10X Genomics' flagship methodology, <em><a href="https://www.10xgenomics.com/products/single-cell-gene-expression" target="_blank">Chromium</a></em>. In brief, the Chromium instrument workflow initiates with a suspension of individual cells or nuclei. Next, leveraging sophisticated microfluidics technology, the instrument meticulously encapsulates single cells or nuclei in individual oil droplets. This process, known as Gel Bead-in-Emulsion (GEM) technology, forms independent microreactors where each of the thousands of individual cells is enveloped within its own microenvironment alongside a gel bead containing reagents for barcoding and capturing genetic material. The optimized workflow that follows enables the construction of high-throughput libraries, preserving the genetic information from these individual cells, and empowering researchers to study cellular heterogeneity and diverse biological mechanisms at an unprecedented single-cell resolution.<br /><br />The Single Cell Genomics workshop at the MSRC was held November 9th, and was sponsored by the <a href="https://inbre.hpcf.upr.edu/" target="_blank">Puerto Rico IDeA Network Biomedical Research Excellence (PR-INBRE)</a> local NIH program. Two of the RIVERA-OMICS undergraduate research assistants, <a href="https://github.com/santiagoDam" target="_blank">Damian Santiago-Mu&ntilde;oz</a> and <a href="https://www.linkedin.com/in/emmanuel-f-6951b2117/" target="_blank">Emmanuel Franceschini-Rosado</a>, attended the event with the primary objective of gaining critical insights into processing single-cell sequencing data and comprehending the evolving landscape of single-cell biology. Attending this workshop provided them with exposure to cutting-edge research techniques and valuable training opportunities, positioning them to integrate new methodologies into their lab work and stay abreast of advancements in the field. Of note, <a href="https://www.linkedin.com/in/nicol%C3%B2-bellin-a22a3b178/" target="_blank">Dr. Nicol&ograve; Bellin</a>, Computational Biologist at the MSRC, imparted the instructional workshop on how to handle single-cell data and provided the skeleton for an R-based pipeline to the workshop attendees.<br /><hr><br /><blockquote><p>As a computer scientist by training, I'm very motivated to expand my skills into the field of bioinformatics research, and Nicol&ograve;'s workshop served to push me further into embarking on a path towards learning how to process data from cutting edge biotechnologies.</p></blockquote>   <br /><p style="text-align:right;"><span style="font:12px &#34;Hiragino Kaku Gothic Pro&#34;,&#34;ヒラギノ角ゴ Pro W3&#34;,&#34;Osaka&#34;,&#34;ＭＳ Ｐゴシック&#34;,&#34;MS PGothic&#34;, sans-serif; ">&ndash; Damian E. Santiago-Mu&ntilde;oz<br />Bioinformatics Research Trainee<br />RIVERA-OMICS<br /></span><hr><br />After concluding the event at MSRC, the lab received an invitation from <a href="https://www.linkedin.com/in/john-walsh-422497219/" target="_blank">John Walsh</a>, current Southeast Sales Associate of 10X Genomics, to the Single Cell Genomics Symposium at the UPR-CCC, held on November 30th and led by distinguished experts focusing on the latest advancements in cancer research technologies. Upon learning about the symposium, the RIVERA-OMICS team quickly gathered up to discuss research student availability, since visiting the UPR-CCC would definitely prove to be of excellent exposure to the lab. Our lab&rsquo;s Principal Investigator, <a href="https://www.linkedin.com/in/osvaldoriveraphd/" target="_blank">Dr. Rivera</a>, alongside 3 research students, Ian, Damian, and myself (Paola), met up at UAGM Cupey Campus and carpooled to the UPR-CCC the day of the symposium, discussing along the way the 10X Genomics technology and its impact to the scientific community.<br /><br />Upon arrival, we were quickly greeted by <a href="https://www.linkedin.com/in/laurenrowan/" target="_blank">Dr. Lauren Rowan</a>, Sales Executive at 10x Genomics, as well as Mr. Walsh. Additionally, Dr. Rivera quickly sprung conversation with colleague <a href="https://prsciencetrust.discovery.symplectic.org/julie_dutil/grants" target="_blank">Dr. Julie Dutil</a>, current Director of the <a href="https://www.cccupr.org/promic/" target="_blank">Puerto Rico OMIC Center (PROMIC)</a> at the UPR-CCC, a collaborative effort to conglomerate resources aimed at performing state-of-art omics methodologies for investigators at the University of Puerto Rico System (UPR), the UPR-CCC, and colleagues at the public and private institutions. Dr. Dutil is also a leading investigator of breast cancer risk variants across historically underrepresented populations, as well as a PI in partnership grants with the likes of <a href="https://www.moffitt.org/research-science/researchers/julie-dutil/" target="_blank">Moffitt Cancer Center (MCC)</a>, aiming to strengthen the Puerto Rican tumors biobank, and establish a network and infrastructure to reduce health disparities in the treatment of cancer in Hispanic patients.<br /><br />During the inaugural seminar, Dr. Lauren Rowan, Sales Executive at 10x Genomics, highlighted the challenges associated with the limited representation of the Hispanic population in various research areas. She emphasized the 10X Genomics' fundamental mission to address current health disparities by providing innovative and accessible scientific tools to local investigators. Next, Dr. Julie Dutil, discussed the resources of the PROMIC at the UPR-CCC and how they are gearing up to enable 10X Genomics tools for basic science and clinical research in the island. Afterwards, Dr. Mote continued to emphasize 10x Genomics' commitment to accelerating advancements in health in his talk titled: Gain a Multidimensional View of Biology with Single Cell Solutions. The seminar focused on the Chromium X/iX, enabling single-cell analysis applicable in oncology, immunology, and neurosciences, with the iX marketed as a lower-cost alternative to facilitate entry into pilot studies and experiments with individual cells. It is a noteworthy approach, since iX is the same instrument as the X, limited only in assay throughput which can be addressed by a simple software update. <br /><br /></p><p><img class="imageStyle" alt="CCC-UPR-10X-SYMP" src="ccc-upr-10x-symp.jpg" width="2000" height="1125" /><br /><span style="font:12px &#34;Hiragino Kaku Gothic Pro&#34;,&#34;ヒラギノ角ゴ Pro W3&#34;,&#34;Osaka&#34;,&#34;ＭＳ Ｐゴシック&#34;,&#34;MS PGothic&#34;, sans-serif; color:#7F8080;">Photo: The Auditorium at the UPR-CCC was packed for the the Single Cell Genomics Symposium. At the podium is </span><span style="font:12px &#34;Hiragino Kaku Gothic Pro&#34;,&#34;ヒラギノ角ゴ Pro W3&#34;,&#34;Osaka&#34;,&#34;ＭＳ Ｐゴシック&#34;,&#34;MS PGothic&#34;, sans-serif; color:#7F8080;"><a href="https://www.linkedin.com/in/ryansmotephdmba/" target="_blank">Dr. Ryan Mote</a></span><span style="font:12px &#34;Hiragino Kaku Gothic Pro&#34;,&#34;ヒラギノ角ゴ Pro W3&#34;,&#34;Osaka&#34;,&#34;ＭＳ Ｐゴシック&#34;,&#34;MS PGothic&#34;, sans-serif; color:#7F8080;">, Science Technology Advisor at 10X Genomics, is imparting the leading seminar titled: "Gain a Multidimensional View of Biology with Single Cell Solutions" .</span><br /><hr><br />The 10X Genomics portfolio of instruments would be further clarified, by <a href="https://www.linkedin.com/in/leilanimartysantos/" target="_blank">Dr. Leilani Marty-Santos</a>, in a session that delved into spatial biology using Visium Spatial and Xenium In Situ instruments. These technologies allow for a three-dimensional analysis of the transcriptome in intact tissues, preserving morphology and providing valuable insights into spatial biology. The <a href="https://www.10xgenomics.com/platforms/visium" target="_blank">Visium platform</a> stands out for its purpose in hypothesis generation capabilities, since it&rsquo;s ideally used for transcriptome-wide spatial context. The development team has meticulously ensured mRNA compatibility through rigorous testing for optimal assay performance (mRNA score &le; 7). The instrument, Visium CytAssist, streamlines the transfer of transcriptomic probes to Visium slides, broadening spatial profiling capabilities. Of note, Visium is compatible with both FFPE and fresh stained tissue section slides.<br /><br /><img class="imageStyle" alt="visium-workflow" src="visium-workflow.png" width="3468" height="1190" /><br /><span style="font:12px &#34;Hiragino Kaku Gothic Pro&#34;,&#34;ヒラギノ角ゴ Pro W3&#34;,&#34;Osaka&#34;,&#34;ＭＳ Ｐゴシック&#34;,&#34;MS PGothic&#34;, sans-serif; color:#7F8080;">Figure 1: Visium Spatial Gene Expression Slide has thousands of barcoded capture-areas with lined with millions of probes to prepare a sequencing library with spatial information that can be directly used in standard high-throughput short-read DNA sequencing.  </span><br /><hr><br /><br />On the other hand, the <a href="https://www.10xgenomics.com/instruments/xenium-analyzer" target="_blank">Xenium platform</a> directly identifies RNA targets within tissue sections; this approach offers researchers a detailed gene expression map while upholding the high sub-cellular  resolution of microscopy studies. Using circularizable probes and enzymatic circular rolling amplification, Xenium generates targets for fluorescent probe hybridization in the Xenium analyzer. The instrument then cycles between capturing  fluorescent microscopy images, followed by washing away each probe and reapplying the next, thus revealing the identities of thousands of RNA transcripts across individual cells in a tissue slice. Dr. Marty-Santos unveiled that future enhancements aim for simultaneous detection of RNA and proteins, promising deeper insights into complex expression. <br /><br /><img class="imageStyle" alt="xenium-workflow" src="xenium-workflow.png" width="3466" height="1192" /><br /><span style="font:12px &#34;Hiragino Kaku Gothic Pro&#34;,&#34;ヒラギノ角ゴ Pro W3&#34;,&#34;Osaka&#34;,&#34;ＭＳ Ｐゴシック&#34;,&#34;MS PGothic&#34;, sans-serif; color:#7F8080;">Figure 2: The Xenium platform uses circular padlock-like probes to perform rolling circle amplification and in-situ probe imaging cycles. The analysis software performs high-powered computation on-instrument. Internal image sensor data is converted into ready-to-explore formats as soon your run finishes.  </span><br /><hr><br />Just before lunchtime, <a href="https://www.thefrancolab.org/research" target="_blank">Dr. Héctor Franco</a>, Lead Investigator Division of Clinical and Translational Cancer Research and Co-Director of the Bioinformatics and Biostatistics Core, immersed the audience in the regulatory logic of cancer cells through single-cell multi-omics. Emphasis was placed on the importance of understanding diseases mechanisms of gene accessibility through the lens of single-cell ATAC sequencing. Dr. Franco highlighted how advanced technologies simplified the processes of sample collection, dissociation, and cell treatment, improving research efficiency. Although briefly mentioned, CRISPR technology will be a potential area for future development, especially in integrating genetic editing techniques with 10x Genomics technologies.<br /><br />In summary, the symposium provided an unparalleled opportunity to understand how these cutting-edge platforms could revolutionize research across domains like cancer, immunology, neuroscience, and developmental biology, allowing scientists to explore biology with unprecedented depth and precision. Furthermore, it highlighted the imperative need to address challenges in Hispanic representation within biomedical research. It emphasized the fundamental role played by the innovative technologies introduced by 10x Genomics in reshaping our understanding of cellular and spatial biology. After all the conferences, we were offered the valuable opportunity to interact with experts such as Dr. Héctor Franco and Dr. Julie Dutil. Encouragement was given to maintain ongoing communication for possible collaborative activities, with a brief insightful period allowing for direct assessment of their perspectives within the field.<br /><div class="button-container"><br /><a href="https://www.10xgenomics.com/" class="zoom-button" target="_blank" style="background-image: url('../../resources/10x-banner.png');"><br /><div class="overlay"><br /><h1 class="button-text">Explore single-cell <br> genomics technologies</span><br /></div> </a></div><br /><br /><span style="font:12px &#34;Hiragino Kaku Gothic Pro&#34;,&#34;ヒラギノ角ゴ Pro W3&#34;,&#34;Osaka&#34;,&#34;ＭＳ Ｐゴシック&#34;,&#34;MS PGothic&#34;, sans-serif; "><script type="text/javascript" src="https://platform-api.sharethis.com/js/sharethis.js#property=6576a5d8a6ecd400190b3bb4&product=inline-share-buttons&source=platform" async="async"></script><div class="sharethis-inline-share-buttons"></div></div></span><br /><hr><br /><em>The RIVERA-OMICS lab had a wonderful time at the symposium. It was a great opportunity to learn about new technologies, listen to amazing science, make new connections and briefly reconnect with colleagues.</em><br /><br /><span style="font:12px &#34;Hiragino Kaku Gothic Pro&#34;,&#34;ヒラギノ角ゴ Pro W3&#34;,&#34;Osaka&#34;,&#34;ＭＳ Ｐゴシック&#34;,&#34;MS PGothic&#34;, sans-serif; ">Authored by:<br /><div class="author-section"><br />  <div class="author-image"><a href="/team/"><br />  <img class="zoom-button" src="../../resources/paola_bitmoji_sq.png" alt="Author - Paola Aponte"></a><br />  </div><br />  <div class="author-info"><br /><strong><a href="/team/">Paola N. Aponte-Castro</a></strong><br />Research Trainee<br />  </div><br /></div><br /><div class="author-section"><br />  <div class="author-image"><a href="/team/"><br />    <img  class="zoom-button" src="../../resources/os_gitmoji_sq.png" alt="Author - Osvaldo Rivera"></a><br />  </div><br />  <div class="author-info"><br /><strong><a href="/team/">Osvaldo D. Rivera-Gonzalez</a></strong><br />Principal Investigator<br />  </div><br /></div><br /><br /></span><span style="font:12px HiraKakuPro-W6; font-weight:bold; font-weight:bold; ">About RIVERA-OMICS:</span><span style="font:12px &#34;Hiragino Kaku Gothic Pro&#34;,&#34;ヒラギノ角ゴ Pro W3&#34;,&#34;Osaka&#34;,&#34;ＭＳ Ｐゴシック&#34;,&#34;MS PGothic&#34;, sans-serif; "><br />OMICS (Omics Molecular Innovations and Commercialization Strategies), is an entity built to develop omics-focused academic research projects and investigate the commercial landscape of novel biotechnologies and biomedicines. OMICS is a branch of </span><span style="font:12px &#34;Hiragino Kaku Gothic Pro&#34;,&#34;ヒラギノ角ゴ Pro W3&#34;,&#34;Osaka&#34;,&#34;ＭＳ Ｐゴシック&#34;,&#34;MS PGothic&#34;, sans-serif; "><a href="https://riverainc.org/" target="_blank">RIVERA Initiatives Inc</a></span><span style="font:12px &#34;Hiragino Kaku Gothic Pro&#34;,&#34;ヒラギノ角ゴ Pro W3&#34;,&#34;Osaka&#34;,&#34;ＭＳ Ｐゴシック&#34;,&#34;MS PGothic&#34;, sans-serif; ">, a non-profit organization that seeks to empower academia through comprehensive resources and collaborative research endeavors to advance educational excellence and innovation.<br /><br />For inquiries, please contact:<br /></span><span style="font:12px &#34;Hiragino Kaku Gothic Pro&#34;,&#34;ヒラギノ角ゴ Pro W3&#34;,&#34;Osaka&#34;,&#34;ＭＳ Ｐゴシック&#34;,&#34;MS PGothic&#34;, sans-serif; "><a href="mailto:research@riveraomics.org" target="_blank">research@riveraomics.org</a></span><span style="font:12px &#34;Hiragino Kaku Gothic Pro&#34;,&#34;ヒラギノ角ゴ Pro W3&#34;,&#34;Osaka&#34;,&#34;ＭＳ Ｐゴシック&#34;,&#34;MS PGothic&#34;, sans-serif; "><br /></span></p></div></div>
	</div>
	

                    </div>

                    <div class="col-sm-12 sidebar">
                        <h2></h2>
                         <div id="blog-categories"><a href="category-osvaldo-d.-rivera-gonz00e1lez.html" class="blog-category-link-enabled">Osvaldo D. Rivera-González (3)</a><br /><a href="category-paola-n.-aponte-castro.html" class="blog-category-link-enabled">Paola N. Aponte-Castro (2)</a><br /><a href="category-sebasti00e1n-a.-rosario-torres.html" class="blog-category-link-enabled">Sebastián A. Rosario-Torres (1)</a><br /><a href="category-ver00f3nica-defill00f3-martinez.html" class="blog-category-link-enabled">Verónica Defilló-Martinez (1)</a><br /><a href="category-emmanuel-franceschini-rosado.html" class="blog-category-link-enabled">Emmanuel Franceschini-Rosado (1)</a><br /><a href="category-damian-e.-santiago-mu00f1oz.html" class="blog-category-link-enabled">Damian E. Santiago-Muñoz (1)</a><br /></div><div id="blog-archives"><a class="blog-archive-link-enabled" href="archive-2024.html">2024</a><br /><a class="blog-archive-link-enabled" href="archive-2023.html">2023</a><br /></div><div id="blog-rss-feeds"><a class="blog-rss-link" href="feed.xml" rel="alternate" type="application/rss+xml" title="Lab to Market">Lab to Market</a><br /></div>
                    </div>
                </div>
			</div>
		</section>
	</div>

	<div class="footer">
		<div class="container">
			<div class="row">
				<div class="col">
					<ul class="navbar-nav ml-auto"><li class="nav-item has-dropdown"><a href="../../team/" rel="" class="nav-link dropdown-toggle" data-toggle="dropdown">Team</a></li><li class="nav-item"><a href="../../projects/" rel="" class="nav-link">Research</a></li><li class="nav-item"><a href="../../publications/" rel="" class="nav-link">Publications</a></li><li class="nav-item has-dropdown"><a href="../../protocols/" rel="" class="nav-link dropdown-toggle" data-toggle="dropdown">Protocols</a></li><li class="nav-item"><a href="https://github.com/riveraomics" target="_blank" class="nav-link">Github</a></li><li class="nav-item has-dropdown"><a href="../../intranet/" rel="" class="nav-link dropdown-toggle" data-toggle="dropdown">Intranet</a></li><li class="nav-item active"><a href="../" rel="" class="nav-link">Blog</a></li></ul>
					 <hr style="margin-top: 5px; margin-bottom: 5px;"> <br> 2023 - 2024 &copy; RIVERA-OMICS <br> <a href="https://www.linkedin.com/company/rivera-omics" target="_blank"><img src="../../resources/linkedin-button.png" alt="LinkedIn" class="custom-social-icon"></a> <a href="https://twitter.com/ozcoscientific" target="_blank"><img src="../../resources/x-twitter-button.png" alt="X-Twitter" class="custom-social-icon"></a> <br> <span style="font-size: 14px; color: white;"> <br><a href="https://riverainc.org" target="_blank"><img src="../../resources/rivera_logo_wt.png" class="responsive-footer-image" alt="RIVERA inc"></a><br> a branch of <a href="https://riverainc.org" target="_blank" style="color: white; text-decoration: underline;">RIVERA Initiatives Inc.</a></span> <br> <span span style="font-size: 14px;">Website designed by <a href=https://ozcorp.io target="_blank"> Ozcorp Scientific </a> </span>
				</div>
			</div>
		</div>
	</div>
	<script type="text/javascript" src="../../rw_common/themes/Future/js/main.js?rwcache=730756410"></script>
</body>

</html>